geo/Cargo.toml

```diff
@@ -18,6 +18,8 @@ travis-ci = { repository = "georust/geo" }
 num-traits = "0.2"
 serde = { version = "1.0", optional = true, features = ["derive"] }
 rstar = { version = "0.7" }
+geographiclib-rs = { version = "0.1", git = "https://github.com/michaelkirk/geographiclib-rs" }
```

Comment:
I don't want to merge this repo reference as is. 

My preference would be moving the geographiclib-rs repo into the georust org and publishing from there. Thoughts?

Reply:
/cc @Psykopear

Reply:
if you get @Psykopear's approval, +1 to move into the georust org. let me know if you need permissions for that

Reply:
Hi @frewsxcv and @michaelkirk 
The code is released under MIT License on purpose, so please take whatever you need and use it how you please.
I just had an email exchange with Michael, with the work he did you can consider him an author as much as me. As I said to him I probably won't be working on this for a while.